test/integration/suites/datastore-mysql-replication/01-test-variants

```diff
@@ -118,5 +118,11 @@ test-mysql-replication() {
     docker-stop "${readwrite_service_name}" "${readonly_service_name}"
 }
 
-test-mysql-replication mysql-5-7 || exit 1
+arch=$(uname -m)
```

Comment:
If you'd like to dedup this logic, the `test/integration/suites/common` file is sourced into each script before it is executed. You could add a function there.

Reply:
Makes sense. This would also affect the approach in https://github.com/spiffe/spire/pull/3884. I can do this as a follow-up PR so we can try to get all the integration test fixes in before 1.6.0.

Reply:
sgtm!